src/Blocks/BlockPressurePlate.h

```diff
@@ -28,7 +29,8 @@ class cBlockPressurePlateHandler :
 			return false;
 		}
 
-		return (cBlockInfo::FullyOccupiesVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)));
+		return (cBlockInfo::FullyOccupiesVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)) ||
+			cBlockFenceHandler::IsFenceBlockType(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)) );
```

Comment:
Bad formatting. When breaking up long conditions, also break the first item onto a separate line; and don't use spaces in front of closing parentheses:
```cpp
return (
  LongCond1() ||  // optional comment about LongCond1
  LongCond2()     // optional comment about LongCond2
)